res/values/themes.xml

```diff
@@ -203,4 +203,56 @@
     <item name="emoji_button_bg">?colorKeyActivated</item>
     <item name="emoji_color">@android:color/system_accent1_900</item>
   </style>
+	<style name="PastelDark" parent="@style/BaseTheme">
+    <item name="android:isLightTheme">false</item>
+    <item name="colorKeyboard">#333334</item>
+    <item name="colorKey">#11111B</item>
+    <item name="colorKeyActivated">#22222A</item>
+    <item name="colorLabel">#CDD6F4</item>
+    <item name="colorLabelActivated">#ffffff</item>
+    <item name="colorLabelLocked">#8AADF4</item>
+    <item name="colorSubLabel">#DEE5E3</item>
+    <item name="keyBorderWidth">0dip</item>
+    <item name="keyBorderWidthActivated">0dip</item>
+    <item name="keyBorderColorLeft">#8AADF4</item>
+    <item name="keyBorderColorTop">#8AADF4</item>
+    <item name="keyBorderColorRight">#8AADF4</item>
+    <item name="keyBorderColorBottom">#8AADF4</item>
+    <item name="emoji_button_bg">?colorKeyActivated</item>
+    <item name="emoji_color">#000000</item>
+</style>
+<style name="PastelLight" parent="@style/BaseTheme">
+		<item name="android:isLightTheme">true</item>
+		<item name="colorKeyboard">#E6E9EF</item>
+		<item name="colorKey">#DCE0E8</item>
+		<item name="colorKeyActivated">#EFF1F5</item>
+		<item name="colorLabel">#4C4F69</item>
+		<item name="colorLabelActivated">#9CA0B0</item>
+		<item name="colorLabelLocked">#437FF7</item>
+		<item name="colorSubLabel">#EFF1F5</item>
```

Comment:
Corner labels are too bright to be visible on a bright background (center labels are dark).